main.js

```diff
@@ -3,46 +3,49 @@ saveBoxer.addEventListener('click', createBoxer);
 // const editBoxer = document.querySelector('#editBoxer');
 // editBoxer.addEventListener('click', editBoxer);
 
+
 function createBoxersArray() {
-  const newBoxer = [
-    firstName.value,
-    lastName.value,
-    nationality.value,
-    weightClass.value,
-  ]
-  localStorage.setItem('boxers', JSON.stringify(newBoxer));
-}
+  const boxers = [];
+  const newBoxer = {
+    firstName: firstName.value,
+    lastName: lastName.value,
+    nationality: nationality.value,
+    weightClass: weightClass.value,
+    wins: 0,
+    losses: 0
+  };
+  boxers.push(newBoxer);
+  localStorage.setItem('boxers', JSON.stringify(boxers));
+};
 
 function updateBoxersArray() {
-  // debugger;
-  const newBoxer = [
-    firstName.value,
-    lastName.value,
-    nationality.value,
-    weightClass.value,
-  ];
-  const previousBoxers = JSON.parse(localStorage.getItem('boxers'));
-  const tempBoxers = [...previousBoxers];
-  newBoxer.push(...tempBoxers);
-  localStorage.setItem('boxers', JSON.stringify(newBoxer));
-}
+  const newBoxer = {
+    firstName: firstName.value,
+    lastName: lastName.value,
+    nationality: nationality.value,
+    weightClass: weightClass.value,
+    wins: 0,
+    losses: 0
+  };
+  const boxers = JSON.parse(localStorage.getItem('boxers'));
```

Comment:
you really need to get a single function to do this for you mate! You keep repeating this bit of code over and over again.

Reply:
Yes, I noted that as I have gone on through, I keep needing to do this so will create a function to do this.